root/programs/C#/Frameworks/Infrastructure/Public/IO/SymmetricCryptography.cs

```diff
@@ -432,13 +438,15 @@ private static SymmetricAlgorithm CreateSymmetricAlgorithm(EnumSymmetricAlgorith
             SymmetricAlgorithm sa = null;
 
             // AesCryptoServiceProvider, AesManagedは3.5からの提供。
-
-            if (esa == EnumSymmetricAlgorithm.AesCryptoServiceProvider)
-            {
-                // AesCryptoServiceProviderサービスプロバイダ
-                sa = AesCryptoServiceProvider.Create(); // devps(1703)
-            }
-            else if (esa == EnumSymmetricAlgorithm.AesManaged)
+            // ポリシーに合わせて、AesCryptoServiceProviderを削除。
```

Comment:
@daisukenishino 

About AesCryptoServiceProvider class, [MSDN](https://msdn.microsoft.com/ja-jp/library/system.security.cryptography.aescryptoserviceprovider(v=vs.110).aspx) describes as follows:
```txt
Performs symmetric encryption and decryption using the Cryptographic Application Programming Interfaces (CAPI) implementation of the Advanced Encryption Standard (AES) algorithm.
```

Since AesCryptoServiceProvider class uses *CAPI*, I suggest that you add *CAPI* to policy.
- CAPI was replaced with CNG on Windows Vista or later.
- Before Windows Vista (e.g. Windows XP) support has ended.
- So, AesCryptoServiceProvider class, that uses *CAPI*, was deleted.